render/egl.c

```diff
@@ -250,26 +265,31 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display,
 			"eglQueryWaylandBufferWL");
 	}
 
+	const char *client_exts_str = eglQueryString(EGL_NO_DISPLAY, EGL_EXTENSIONS);
```

Comment:
Instead of querying again the client exts, I'd prefer to add a new `bool device_query_ext` field to `wlr_egl.exts`.

Reply:
I also moved the output if the `client_exts_str` from `egl_init` to `egl_create`, let me know if you want something else.

Reply:
Yes, perfectly fine by me.